### PR TITLE
Add --skip-ssl option to post install to skip auto generation of SSL …

### PR DESCRIPTION
**Pull request by limited**
…certificate

This fixes https://github.com/Comcast/traffic_control/issues/1179

### CODE REVIEW

File: traffic_ops/install/bin/postinstall [hunk 1]
```diff
@@ -48,6 +49,10 @@ my $tmAdminUser = "";
 my $tmAdminPw = "";
 my $parameters;
 
+my $skip_ssl = 0;
+GetOptions("skip-ssl" => \$skip_ssl)
+or die ("Error in command line arguments");
+
```
Comment (dangogh) [inline traffic_ops/install/bin/postinstall]: indentation issues both here and below..  perltidy?

File: traffic_ops/install/bin/postinstall [hunk 2]
```diff
@@ -48,6 +49,10 @@ my $tmAdminUser = "";
 my $tmAdminPw = "";
 my $parameters;
 
+my $skip_ssl = 0;
+GetOptions("skip-ssl" => \$skip_ssl)
```
Comment (dangogh) [inline traffic_ops/install/bin/postinstall]: how about adding a --help?   No way to tell what the option is without looking thru the source..

**Comment by limited** [comment]
I'll fix, need a day or so

**Comment by limited** [comment]
perltidy went crazy on me here- any suggestions on how to adjust @dangogh ? I copy and pasted the lines as is from the traffic-control-cdn developers page

**Comment by dangogh** [comment]
@limited this should be covered by #126,  don't you think?

**Comment by limited** [comment]
Not yet, but Peter suggested adding a question to the input in 126 to allow a user to skip it. I would support that approach over this PR.

**Comment by dangogh** [comment]
I think this should be closed -- postinstall now has `-defaults` option to write out a cfg file with all default questions/answers.   And `-a -cfile=<file>` to load all from the file with no user input.

To skip cert generation,  change this answer to `"no"`:
```
   "/opt/traffic_ops/install/data/json/openssl_configuration.json" : [
      {
         "Do you want to generate a certificate?" : "yes",
         "config_var" : "genCert"
      },
```

**Comment by dneuman64** [comment]
@limited I think this can be closed per @dangogh's comment above

**Pull request closed without merging**